aag/weather.py

```diff
@@ -748,9 +748,11 @@ def calculate_and_set_pwm(self):
 
                 self.set_pwm(new_pwm)
 
-    def make_safety_decision(self, current_values):
+    def make_safety_decision(self, current_values, ignore=False):
         """
         Method makes decision whether conditions are safe or unsafe.
+
+        ignore: list of saftey params to ignore. Can be 'rain', 'wind', 'gust' or 'cloud'
```

Comment:
American english please :)

```suggestion
        ignore: list of safety params to ignore. Can be 'rain', 'wind', 'gust' or 'cloud'. If None (default) nothing is ignored.
```